automation/jleechanorg_pr_automation/jleechanorg_pr_monitor.py

```
@@ -1352,6 +1417,188 @@ def check_chrome_cdp_accessible(port=9222, host="127.0.0.1", timeout=5):
+        for candidate in mac_candidates:
+            if Path(candidate).exists():
+                return candidate
+
+    for command in ("google-chrome", "google-chrome-stable", "chromium", "chromium-browser"):
+        found = shutil.which(command)
+        if found:
+            return found
+    return None
+
+
+def _start_chrome_debug(port: int, user_data_dir: str) -> Tuple[bool, str]:
```

Ruff (0.14.10)

Too many return statements (10 > 6)

(PLR0911)